tetris.py

```diff
@@ -25,16 +28,32 @@
 
 GameStatus = Enum("GameStatus", "in_progress, game_lost, paused")
 
+try:
+    image_dir = pathlib.Path(sys._MEIPASS)
+except AttributeError:
+    image_dir = pathlib.Path(__file__).parent / "images"
```

Comment:
The latest minesweeper has comments explaining what this code does. Please add them here too. They are needed. I explained the code to ThePhilgrim at least 2 times before we added the comments.